### DISCUSSION TITLE
Résultats de la compétition

### DISCUSSION THREAD

**Discussion by beng-git** [danymat/INSAnonym-utils#106] [General]
Bonjour,

Un grand merci et toutes nos félicitations à tous les participants, malgré les problèmes techniques de métriques, l'ensemble des participants ont pu au moins soumettre un jeu de données anonymisé et une attaque. Nous espérons que vous aurez apprécié ce challenge, et nous vous invitons à répondre à ce questionnaire pour noter votre avis sur la compétition, et comment l'améliorer : https://forms.gle/RrWKXS1RiAax99Gi6 

Classement défense : 

1- DAZC 18.42%
2- THAThack 2.9%
3- THE 1.73%
4- anonym212 1.56%
5- pocochocambo 0.76%
6- 404 0.63%
7- VietRoc 0.46%
8 ex- KodaTeam & MichelLardon 0.16%
10- autofill, OSS, Sybil, TresAnonyme, ServiceInactifAnonyme

Classement attaque : 

1- VietRoc 12.15
2- anonym212 10.70
3- pocochocambo 9.96
4- Sybil 9.03
5- THAThack 7.77
6- DAZC 7.66
7- MichelLardon 4.62
8- autofill 3.28
9- TresAnonyme 2.18
10- ServiceInactifAnonyme 2.02
11- THE 1.00
12- KodaTeam 0.94
13- OSS 0.04
14- 404 0.0

Cordialement,

l'équipe DARC

**Comment by ceichler** [comment]
Félicitations à toutes les équipes
